tests/SeedTest.php

```diff
@@ -161,16 +161,37 @@ public function testMerge5()
 
         $this->assertSame($o, $oo);
         $this->assertEquals($oo->foo, 'xx');
+    }
 
+    public function testMerge5b()
+    {
         // and even if multiple objects are found
-        $o = new SeedDITestMock();
+        $o = new ViewTestMock();
         $o->foo = ['red'];
-        $o2 = new SeedDITestMock();
+        $o2 = new ViewTestMock();
         $o2->foo = ['yellow'];
         $oo = $this->mergeSeeds(['foo'=>['xx']], $o, ['foo'=>['green']], $o2, ['foo'=>['cyan']]);
 
         $this->assertSame($o, $oo);
-        $this->assertEquals($oo->foo, ['red', 'xx']);
+        $this->assertEquals(['red', 'xx'], $oo->foo);
+    }
+
+    public function testMerge6()
+    {
+        $oo = $this->mergeSeeds(['4'=>'four'], ['5'=>'five']);
+        $this->assertEquals($oo, ['4'=>'four', '5'=>'five']);
+
+        $oo = $this->mergeSeeds(['4'=>['four']], ['5'=>['five']]);
```

Comment:
this test is duplicated above

Reply:
no, the value is array, above value is string.